notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    text = processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
</pre><p>Why do you apply chat template outside VLMPipeline?</p><p>is it not possible to apply chat template inside VLMPipeline?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>